src/pages/LoginPage.tsx

```diff
@@ -32,12 +32,13 @@ const LoginPage: React.FC = () => {
             className={`${classes["login-page__btn"]} ${
               !isSignIn ? classes["inactive"] : ""
             }`}
+            aria-label="Sign Up"
           >
             Sign Up
           </button>
         </div>
       </div>
-      {isSignIn ? <SignIn /> : <SignUp />}
+      <AuthForm isSignIn={isSignIn} setIsSignIn={setIsSignIn} />
```

Comment:
Always have a mindset of do i need to prop down something. In this case, I would like to have this isSignIn in a hook, and when user is switching between two states, to change the url. Adding hash to it (e.g. `#signup`) and that is our state.